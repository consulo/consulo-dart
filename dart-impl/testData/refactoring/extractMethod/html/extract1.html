<html>
<body>
<script type="application/dart">
    main() {
        <selection>var i = 0;
        var j = i + 239;
        i = j - 1;</selection>
        return i + 10;
    }
</script>
</body>
</html>
