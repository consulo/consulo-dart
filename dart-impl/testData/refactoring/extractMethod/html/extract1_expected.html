<html>
<body>
<script type="application/dart">
    int extracted() {
        var i = 0;
        var j = i + 239;
        i = j - 1;
        return i;
    }

    main() {
        var i = extracted();
        return i + 10;
    }
</script>
</body>
</html>
